src/components/global/mainBody/index.js

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import React from 'react';
+import { Box } from '@mui/material';
+// import styled from 'styled-components';
+import CategoryCard from '@/components/global/card';
+import carro2 from '/public/img/carro2.jpg';
+
+
+function MainBody () {
+    return (
+        <Box sx={{
+            backgroundColor: '#fcba03',
```

Comment:
Não precise usar `color` na sintaxe do background, pode deixar apenas background

Coloque o import do theme, e troque a cor do background por
```suggestion
            background: 'theme.palette.background.gradient',
```